index.Rmd

```diff
@@ -67,7 +67,7 @@ Rscript ./script
 python3 ./script
 ```
 
-This allows you to run script files even without having any graphical user interface installed or running, which is the fastest way to run any script from top to bottom. You can use this method to try out the code examples below, or use your favourite integrated development environment (IDE), such as RStudio or RKWard for R and Spyder or Jupyter for Python.
+This allows you to run script files even without having any graphical user interface installed or running, which is the fastest way to run any script from top to bottom. You can use this method to try out the code examples below, or use your favorite integrated development environment (IDE), such as RStudio or RKWard for R and Spyder or Jupyter for Python.
```

Comment:
No, the original spelling "favourite" is correct, make sure to use the British English spell checker ;)